### PR TITLE
add example task topic

### PR DESCRIPTION
**Pull request by taroth21**
First draft of a task topic type.

Open issues:

- metadata?
- article uses no profiling ATM
- productname entity not set yet (on purpose)
- no real links in 'Related topics'
- 'Troubleshooting' section still empty
- not verified for different product versions yet (default base modules etc.)

### CODE REVIEW

File: xml/task-register-product-after-installation.xml [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+     You have installed your product and your system is up and running.
+     </para>
+   </listitem>
+   <listitem>
+    <para>
+     If you want to register your product at the &scc; center, your system needs to be connected to the Internet.
+    </para>
+   </listitem>
+   <listitem>
+    <para>If you want to register your product at an &smt; server, the &rmt; server needs to be
```
Comment (fsundermeyer) [inline xml/task-register-product-after-installation.xml:46]:
```suggestion
    <para>If you want to register your product at an &rmt; server, the &rmt; server needs to be
```

File: xml/task-register-product-after-installation.xml [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+  <title>Requirements</title>
+  <itemizedlist>
+   <listitem>
+    <para>
+     You have installed your product and your system is up and running.
+     </para>
+   </listitem>
+   <listitem>
+    <para>
+     If you want to register your product at the &scc; center, your system needs to be connected to the Internet.
```
Comment (fsundermeyer) [inline xml/task-register-product-after-installation.xml:42]:
```suggestion
     If you want to register your product at the &scc;, your system needs to be connected to the Internet.
```

File: xml/task-register-product-after-installation.xml [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+     </mediaobject>
+    </informalfigure>
+    <para> Choose <guimenu>Accept</guimenu> and <guimenu>Finish</guimenu> to conclude the
+     registration process. </para>
+   </step>
+  </procedure>
+ </section>
+
+ <section xml:id="commandline-register-product-after-installation">
+   <title>Registering your product from command line after the installation</title>
```
Comment (fsundermeyer) [inline xml/task-register-product-after-installation.xml:166]:
```suggestion
   <title>Registering your product from command the line after the installation</title>
```

File: xml/task-register-product-after-installation.xml [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+  <para>To register the base product plus modules and extensions from the command line, use
```
Comment (fsundermeyer) [inline xml/task-register-product-after-installation.xml:167]:
```suggestion
  <para>To register the base product plus its default modules and extensions from the command line, use
```

File: xml/task-register-product-after-installation.xml [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
+     </para>
+    </step>
+   </procedure>
+  </section>
+
+ <section xml:id="summary-register-product-after-installation">
+ <title>Summary</title>
+ <para>
+  After successful registration of a product, the repositories required
+  to keep your product up-to-date have been added to your system.
```
Comment (fsundermeyer) [inline xml/task-register-product-after-installation.xml:211]: The system is now also eligible for support.

File: xml/task-register-product-after-installation.xml [hunk 6]
```diff
@@ -0,0 +1,246 @@
...
+ <para></para>
+</section>
+
+ <section xml:id="next-register-product-after-installation">
+ <title>Next steps</title>
+ <itemizedlist>
+  <listitem>
+   <para>Keeping the system up-to-date</para>
+  </listitem>
+ </itemizedlist>
```
Comment (fsundermeyer) [inline xml/task-register-product-after-installation.xml:226]:
```suggestion
   <listitem>
    <para>Managing modules and extensions in a running system</para>
   </listitem>
  </itemizedlist>

File: xml/task-register-product-after-installation.xml [hunk 7]
```diff
@@ -0,0 +1,246 @@
...
+ <title>Summary</title>
+ <para>
+  After successful registration of a product, the repositories required
+  to keep your product up-to-date have been added to your system.
+ </para>
+</section>
+
+ <section xml:id="troubleshooting-register-product-after-installation">
+ <title>Troubleshooting</title>
+ <para></para>
```
Comment (fsundermeyer) [inline xml/task-register-product-after-installation.xml:217]: Possible causes of a failed registration:
* faulty internet/network connection
* wrong email address

FAQ:
I have added a module/extension and it asks for a registration key

**Review by fsundermeyer — CHANGES_REQUESTED** [review]
Looks good! Thanks a lot!

File: xml/task-register-product-after-installation.xml (hunk 2 above)
Comment (taroth21) [reply xml/task-register-product-after-installation.xml:42]: Nope, I double-checked: `&scc;` already resolves to 'SUSE Customer Center', so rejecting this suggestion.

File: xml/task-register-product-after-installation.xml [hunk 8]
```diff
@@ -0,0 +1,246 @@
...
+     </para>
+    </step>
+   </procedure>
+  </section>
+
+ <section xml:id="summary-register-product-after-installation">
+ <title>Summary</title>
+ <para>
+  After successful registration of a product, the repositories required
+  to keep your product up-to-date have been added to your system.
```
Comment (taroth21) [inline xml/task-register-product-after-installation.xml:211]:
```suggestion
  to keep your product up-to-date have been added to your system. The system is now eligible for support. 
```

**Review by fsundermeyer — APPROVED** [review]
LGTM

**Comment by taroth21** [comment]
Thanks for reviewing and approving, Frank! As most of the open issues listed in the PR description have been addressed meanwhile, I will merge it now.

**Pull request merged**